packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3173,6 +3177,25 @@ class SemanticsConfiguration {
     _onSetSelection = value;
   }
 
+  /// The handler for [SemanticsAction.setText].
+  ///
+  /// This handler is invoked when the user either wants to replace the current
+  /// text in the text field with a new text.
+  ///
+  /// Voice access users can trigger this handler by speaking "type <text>" to
+  /// their Android devices.
+  SetTextHandler? get onSetText => _onSetText;
+  SetTextHandler? _onSetText;
+  set onSetText(SetTextHandler? value) {
+    assert(value != null);
```

Comment:
Nit: Could you remove this assert and make the `value` parameter's type non-nullable?

Reply:
The setter and getter need to match, dart won't let you make a non-nullable with nullable getter.